kotlin/mobilesdkrs/build.gradle

```diff
@@ -83,7 +85,7 @@ tasks.register('bindGen', Exec) {
     workingDir "../../"
     if (Os.isFamily(Os.FAMILY_WINDOWS)) {
         commandLine("cmd", "/c",
-                "cargo build && " + "cargo run --bin uniffi-bindgen generate --library target\\debug\\libmobile_sdk_rs.dll " + "--language kotlin " + "--out-dir " + outDir.replace('/', '\\'))
+                "cargo build --release && " + "cargo run --bin uniffi-bindgen generate --library target\\release\\libmobile_sdk_rs.dll " + "--language kotlin " + "--out-dir " + outDir.replace('/', '\\'))
```

Comment:
Any reason to use `debug` instead of `release` here? Safe to update?

Reply:
I'm not a gradle expert. If this is only used for release builds then yes, but we don't want to do optimisations on debug builds.